### PR TITLE
VACMS-4054: Centralized Content Last Updated Date

### PR DESCRIPTION
**Pull request by kaise-lafrai**
## Description

See _issueid_. #4054 

## QA steps

As user _uid_ with _user_role_
1. Do this
1. Then that
1. Then validate Acceptance Criteria from issue
- [ ] This
- [ ] That
- [ ] The other thing

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Core Application Team`
- [ ] `Product Support Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: config/sync/core.date_format.short.yml [hunk 1]
```diff
@@ -7,4 +7,4 @@ _core:
 id: short
 label: 'Default short date'
 locked: false
-pattern: 'm/d/Y - H:i'
+pattern: 'M d Y g:i a'
```
Comment (swirtSJW) [inline config/sync/core.date_format.short.yml:10]: Did @kevwalsh sign off on this change?  Not sure where else short date is used or if it matters.

File: config/sync/core.entity_form_display.node.vet_center.default.yml [hunk 2]
```diff
@@ -71,7 +71,6 @@ third_party_settings:
     group_editorial_workflow:
       children:
         - moderation_state
-        - revision_log
```
Comment (swirtSJW) [inline config/sync/core.entity_form_display.node.vet_center.default.yml:74]: Don't want to lose this.

**Review by swirtSJW — CHANGES_REQUESTED** [review]
Overall looks great Kaise.  There are just a couple of losses of those pesky revision_log and one thing I want to double check with @kevwalsh

**Comment by swirtSJW** [comment]
@kevwalsh  and @rachel-kauff   
One question on this
image

File: config/sync/core.date_format.short.yml (hunk 1 above)
Comment (swirtSJW) [reply config/sync/core.date_format.short.yml:10]: Looks like @rachel-kauff had some concerns about this new format that she left in the issue.

**Comment by rachel-kauff** [comment]
@swirtSJW Great question. I assume there are two dates because those are two discrete pieces of data, but if they're always  going to be the same then yes, let's just show one updated link.

File: config/sync/core.date_format.short.yml (hunk 1 above)
Comment (swirtSJW) [reply config/sync/core.date_format.short.yml:10]: Looks like the old is preferable to the new format  From Kauffe's comment in the issue

> That is to say I'm in favor of using mm/dd/yyyy - H:m

**Comment by kaise-lafrai** [comment]
@rachel-kauff @swirtSJW Thanks for the input. Let me get started!

**Comment by kaise-lafrai** [comment]
@rachel-kauff @swirtSJW Ready for another look. Thanks :)

**Review by swirtSJW — APPROVED** [review]
Looks good to me.  Nice work Kaise.
image

Just waiting on tests to complete.

**Comment by swirtSJW** [comment]
image

**Pull request merged**